docs/manual-test.md

```diff
@@ -232,3 +232,44 @@ Create a new workbook on Tableau Server with the connector installed on the serv
 
     **Note:** Web authoring (creating a new connection from the web) is not currently available for all connector superclasses. In those cases, your connector won't appear on the list of connectors on Tableau Server. If you can publish a workbook or data source using your connector to your server, then your connector is loaded correctly, even if you can't see it on the list of connectors.
 
+## Test an OAuth Connector
+
+If your connector supports OAuth Authentication, besides the previous test steps, there are some extra steps you need to verify for OAuth on Tableau Server.
```

Comment:
Should we also have a specific Desktop test as well? Just a simple case where they create a workbook on desktop using OAuth. 

While the publish scenario technically covers desktop, it'd be nice to have a desktop-only case that they go through first, so that if there are any OAuth problems they can identify and fix it in a context that is easier to understand and set up. (Also, we can't assume that the generic desktop manual test cases will cover OAuth since in multi-auth connectors people might use a different auth type).